src/compiler/crystal/command/format.cr

```diff
@@ -68,7 +68,7 @@ class Crystal::Command
           when .invalid_byte_sequence?
             error "'#{result.filename}' is not a valid Crystal source file", exit_code: nil
           when .bug?
-            error "there's a bug formatting '#{result.filename}', please report it including the contents of the file: https://github.com/crystal-lang/crystal/issues", exit_code: nil
+            error "there's a bug formatting '#{result.filename}', to show more information, please run:\n\n  $ crystal tool format '#{result.filename}'", exit_code: nil
```

Comment:
> For the multiple-backtrace situation i would expect the formatter to report the first error the same as the single-file mode and then exit.

This behavior has a big problem: it makes hard to format all files when formatter bug is found. It should continue to format file as possible.